tasks/4_stand-in-line/stand-in-line.js

```diff
@@ -0,0 +1,4 @@
+export function nextInLine(arr, item) {
```

Comment:
Please follow "pure function" pattern
https://en.wikipedia.org/wiki/Pure_function

Reply:
> Please follow "pure function" pattern
> https://en.wikipedia.org/wiki/Pure_function

I can' use it because we need to change array in arguments. We have test to check that array has changed;